force-app/main/default/classes/SOQL.cls

```diff
@@ -117,7 +144,7 @@ public with sharing class SOQL {
      * @return A list of Case records.
      */
     public static List<Case> getFilterCaseOrigins() {
-        return null; // Replace null with the variable you used to store the result
+        return [SELECT Id, Subject, Description, Origin from Case where Origin NOT IN ('Web', 'Email', '')]; // Replace null with the variable you used to store the result
```

Comment:
No issue here, just something to think about:

One thing I would add about this to think about more long term is scalability.

If you do NOT IN, that is automatically not selective (not optimized by the Salesforce Query Optimizer). You can write better queries if you always think about things you DO want rather than things you DON'T want. E.g., avoid using the NOT operator. So here, you would specify the Origins you DO want to select. The downside to this approach is... you have to know all of the picklist values you do want to select, which might not be business logic. There's a tradeoff here, and ways to do it either way, but something to think about as you work in bigger systems with millions of records.